package.json

```diff
@@ -58,18 +59,25 @@
     "@storybook/node-logger": "^6.1.21",
```

Comment:
Can we remove `react-div-100vh`?

Reply:
The "react-div-100vh" was necessary because we released it as a website, and it's used to accommodate for the browser status bar height changes.
Now that we're releasing an app, it's not necessary anymore.

We just need to make sure it's not used in the coordinator app

Reply:
I think we are still going to release this app in web in the coming weeks.

In such cases, should we leave it for now?

Reply:
I'll re-add it then